Documentation/PackageDescription.md

```diff
@@ -439,13 +444,13 @@ www.semver.org.
 The major version signifies breaking changes to the API which requires
 updating existing clients. For example, renaming an existing type, removing
 a method or changing a method’s signature are considered breaking changes.
-This also includes any backwards incompatible bugfixes or behaviour changes
+This also includes any backwards incompatible bugfixes or behavior changes
```

Comment:
Just for the record, @aciidb0mb3r, I have always liked seeing your commonwealth spelling. It reads so much more smoothly to me north of the 49<sup>th</sup>. (But I realize it is out of place for Silicon Valley.)

Reply:
I like that spelling as well, but both were present in the text, so I changed this one, but I could change them both to be "behaviour" for the international flavour.

Reply:
I am from India so I end up using UK spellings ;) I don't really have a preference but we should be consistent.

Reply:
For reasons of history, it's considered standard throughout software documentation and API naming in general to use American English spellings (even among projects developed chiefly in Europe and other territories using British/International spelling), so that's what we should use here.